producer.py

```diff
@@ -210,6 +190,26 @@ def package_releases_update(
                         package_version,
                         package_index.url,
                     )
+
+                    async_tasks.append(
+                        package_release.publish_to_topic(
+                            package_release.MessageContents(
+                                package_name=package_name,
+                                package_version=package_version,
+                                index_url=package_index.url,
+                                component_name=COMPONENT_NAME,
+                                service_version=__service_version__,
+                            )
+                        )
+                    )
+                    _LOGGER.debug(
+                        "Package %r in version %r hosted on %r was scheduled to be published on topic %r",
```

Comment:
The message sounds misleading to me.

Reply:
Updated in 8d6e76e943232fd50953de72011ec5a33fc561f9